### PR TITLE
Add Romanian translation

### PR DESCRIPTION
**Pull request by adriancostin6**
I added the Romanian translation, tested it and it works properly. I initially had some problems with testing it because I forgot to change my system locale from en_US to Romanian (posted on the IRC chat, without realizing what was the problem). After I fixed that, paru started displaying the text in my native language with my translations.

### CODE REVIEW

**Comment by Morganamilo** [comment]
Jut got home and saw. Was about to reply but realised you left.

File: po/ro.po [hunk 1]
```diff
@@ -0,0 +1,703 @@
...
+msgstr "Există {n} provideri disponibili pentru {pkg}:"
+
+#: src/config.rs:961 src/info.rs:57 src/info.rs:96 src/install.rs:1755
+#: src/install.rs:1804
+msgid "Repository"
+msgstr "Repository"
+
+#: src/config.rs:983
+msgid "  failed to sync {}"
+msgstr " nu s-a putut sincroniza {}"
```
Comment (Morganamilo) [inline po/ro.po:146]: Missing a space

File: po/ro.po [hunk 2]
```diff
@@ -0,0 +1,703 @@
...
+#: src/download.rs:294
+msgid " ({total>padding$}/{total}) downloading: {pkg}"
+msgstr " ({total>padding$}/{total}) se descarcă: {pkg}"
+
+#: src/download.rs:317
+msgid "Downloading PKGBUILDs..."
+msgstr "Se descarcă PKBUILDs..."
+
+#: src/download.rs:321
+msgid " PKGBUILDs up to date"
```
Comment (Morganamilo) [inline po/ro.po:221]: Missing space

File: po/ro.po [hunk 3]
```diff
@@ -0,0 +1,703 @@
...
+
+#: src/install.rs:762
+msgid "failed to read dir: {}"
+msgstr "nu s-a putut citi directorul: {}"
+
+#: src/install.rs:778
+msgid ""
+"{} is a directory\n"
+"\n"
+msgstr "{} este un director\n"
```
Comment (Morganamilo) [inline po/ro.po:421]: Copy the formatting above.

File: po/ro.po [hunk 4]
```diff
@@ -0,0 +1,703 @@
...
+msgid "yes"
+msgstr "yes"
+
+#: src/util.rs:103
+msgid "n"
+msgstr "n"
+
+#: src/util.rs:103
+msgid "no"
+msgstr "no"
```
Comment (Morganamilo) [inline po/ro.po:683]: Shouldn't this be nu?

File: po/ro.po [hunk 5]
```diff
@@ -0,0 +1,703 @@
...
+#: src/clean.rs:31
+msgid "Do you want to remove ALL AUR packages from cache?"
+msgstr "Doriți să stergeți din cache TOATE PACHETELE care provin din AUR?"
+
+#: src/clean.rs:33
+msgid "Do you want to remove all other AUR packages from cache?"
+msgstr "Doriți să stergeți din cache toate celelalte pachete AUR?"
+
+#: src/clean.rs:40
+msgid "Clone Directory: {}"
```
Comment (Morganamilo) [inline po/ro.po:27]: Is there no translation for directory?

File: po/ro.po [hunk 6]
```diff
@@ -0,0 +1,703 @@
...
+msgid "only one operation may be used at a time"
+msgstr "se poate folosi doar o singură operațiune la un anumit moment"
+
+#: src/config.rs:638
+msgid "can not find local repo '{}' in pacman.conf"
+msgstr "nu s-a putut găsi repo-ul local '{}' în pacman.conf"
+
+#: src/config.rs:670
+msgid "failed to initialize alpm: root={} dbpath={}"
+msgstr "nu s-a putut inițializa alpm: root={} dbpath{}"
```
Comment (Morganamilo) [inline po/ro.po:105]: Missing =

File: po/ro.po [hunk 7]
```diff
@@ -0,0 +1,703 @@
...
+msgid "error: unknown option '{}' in section [bin]"
+msgstr "eroare: opțiune necunoscută '{}' în secțiunea [bin]"
+
+#: src/config.rs:891
+msgid "error: unknown option '{}' in section [options]"
+msgstr "erorare: opțiune necunoscută '{}' în secțiunea [options]"
+
+#: src/config.rs:896
+msgid "option '{}' does not take a value"
+msgstr "opțiunea '{}' nu necesită(primește) o valoare"
```
Comment (Morganamilo) [inline po/ro.po:133]: What's the brackets here for?

File: po/ro.po [hunk 8]
```diff
@@ -0,0 +1,703 @@
...
+msgid "failed to get current exe"
+msgstr "nu s-a putut obține current exe"
+
+#: src/repo.rs:195
+msgid "syncing local databases..."
+msgstr "se sincronizează baza de date locală..."
+
+#: src/repo.rs:201
+msgid "  nothing to do"
+msgstr " nimic de făcut"
```
Comment (Morganamilo) [inline po/ro.po:583]: Missing space

Comment (adriancostin6) [reply po/ro.po]: Saw the typo, fixed now, thanks for pointing it out.

File: po/ro.po (hunk 6 above)
Comment (adriancostin6) [reply po/ro.po]: Saw the typo, fixed now, thanks for pointing it out.

File: po/ro.po (hunk 5 above)
Comment (adriancostin6) [reply po/ro.po:27]: I didn't know how to interpret this, but managed to find a decent translation at last. It is ok now, as far as I can tell. This refers to the directory where the cloned files are stored right?

File: po/ro.po (hunk 4 above)
Comment (adriancostin6) [reply po/ro.po]: Yes it should, thank you, I actually missed these when I made a second pass to check my translations. It's fixed now, for both no and yes.

File: po/ro.po (hunk 5 above)
Comment (adriancostin6) [reply po/ro.po:27]: Also fixed the translation for the Diff Directory

File: po/ro.po (hunk 3 above)
Comment (adriancostin6) [reply po/ro.po]: Fixed the formatting, thanks for pointing it out.

File: po/ro.po (hunk 2 above)
Comment (adriancostin6) [reply po/ro.po:221]: Saw the typo, fixed now, thanks for pointing it out.

File: po/ro.po (hunk 1 above)
Comment (adriancostin6) [reply po/ro.po]: Saw the typo, fixed now, thanks for pointing it out.

File: po/ro.po (hunk 7 above)
Comment (adriancostin6) [reply po/ro.po]: I was initially unsure how to translate it properly, but I have figured it out and replaced the translation with a simpler and more concise one.

File: po/ro.po [hunk 9]
```diff
@@ -0,0 +1,705 @@
...
+msgid "could not find all required packages:"
+msgstr "nu s-au putut găsi toate pachetele necesare:"
+
+#: src/install.rs:982
+msgid ""
+"\n"
+"    {missing} (wanted by: {stack})"
+msgstr ""
+"\n"
+"{missing} (dorit de: {stack})"
```
Comment (Morganamilo) [inline po/ro.po:459]: Missing space

File: po/ro.po [hunk 10]
```diff
@@ -0,0 +1,705 @@
...
+#: src/upgrade.rs:181
+msgid "{pkg}: ignoring package upgrade ({old} => {new})"
+msgstr "{pkg}: se ignora upgrade-ul pachetului ({old} => {new})"
+
+#: src/upgrade.rs:334
+msgid "Packages to exclude (eg: 1 2 3, 1-3):"
+msgstr "Pachetele care vor fi excluse (eg: 1 2 3, 1-3):"
+
+#: src/util.rs:80
+msgid "[Y/n]:"
```
Comment (Morganamilo) [inline po/ro.po:664]: Shouldn't all these be d/n?

Comment (adriancostin6) [reply po/ro.po:664]: Fixed, thanks!

File: po/ro.po (hunk 9 above)
Comment (adriancostin6) [reply po/ro.po]: Fixed, thanks!

**Pull request merged**